### ISSUE TITLE
xdotool_mousedown with three fingers

### ISSUE THREAD

**Issue by alperenkaplan** [Hikari9/comfortable-swipe#78]
Thanks for the tool. Can we have an option to send mousedown instead of keysequence when 3 fingers used? I would like to move things and select text, like macs, with three fingers. I am using linux on my mac and very used to have mousedown with three fingers. I believe all you need to to do would be send mousedown when 3 fingers detected and send mouseup when all fingers lifted (so we would not have to use 3 fingers during the whole gesture). I would want to contribute myself but I don't write cpp and didn't understand the code very well.

**Comment by Hikari9** [comment]
Hi `alperenkaplan`, this is interesting. Might be worth a try, I'll have a pull request once it's up.

**Comment by Hikari9** [comment]
Hi @alperenkaplan , I'll be closing this issue as I have merged this new feature to master (see #79).

Mouse down is now possible as of `v1.1.0`. Please update installation with the following:

```bash
git clone https://github.com/Hikari9/comfortable-swipe --branch v1.1.0
yes | bash comfortable-swipe/install
rm -rf comfortable-swipe
```

## How to add 3-finger drag

Add 3-finger drag with this line:
```
echo "hold3 = button1" >> "$(comfortable-swipe config)"
```

Restart comfortable-swipe:
```
comfortable-swipe restart
```

Try dragging a window. Should work like a charm. :rocket:
